backend/items/api.py

```
@@ -50,19 +50,22 @@
+from .services import ItemService, CheckupService, ElasticsearchAgentService
 from datetime import datetime
 from uuid import UUID
 from dotenv import load_dotenv
 import os
 import logging
+import json
 from .addItemAgent import run_agent
 from django.core.exceptions import ValidationError
 import jwt
 from django.conf import settings
 from datetime import datetime
```

Ruff (0.15.2)

[error] Redefinition of unused `datetime` from line 54: `datetime` redefined here

Remove definition: `datetime`

(F811)